workup.gemspec

```diff
@@ -17,14 +17,16 @@ Gem::Specification.new do |spec|
   spec.homepage      = 'https://github.com/cvent/workup'
 
   spec.files         = Dir.glob('{lib,files,exe}/**/*', File::FNM_DOTMATCH)
-  spec.bindir        = 'exe'
-  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.bindir        = 'bin'
```

Comment:
Not sure if this requires omnibus changes, but this made the bundle install work